Mathlib/RingTheory/MvPolynomial/Homogeneous.lean

```diff
@@ -255,6 +255,24 @@ lemma finSuccEquiv_coeff_isHomogeneous {N : ℕ} {φ : MvPolynomial (Fin (N+1))
   simpa [Finset.sum_subset_zero_on_sdiff (g := d.cons i)
     (d.cons_support (y := i)) (by simp) (fun _ _ ↦ rfl), Finset.sum_insert aux, ← h] using hφ hd
 
+-- TODO: develop API for `optionEquivLeft` and get rid of the `[Fintype σ]` assumption
+lemma coeff_isHomogeneous_of_optionEquivLeft_symm
+    [Fintype σ] {p : Polynomial (MvPolynomial σ R)}
```

Comment:
Can this be relaxed to `Finite` in the meantime? Or is it needed by the statement?